web_src/css/repo/list-header.css

```diff
@@ -5,6 +5,10 @@
   gap: .5rem;
 }
 
+.list-header > .ui.button {
+  margin: 0;
+}
+
```

Comment:
It could use `tw-m-0` for these buttons, instead of introducing this one-time purpose style.